recipe/hardcoded-paths.patch

```diff
@@ -16,7 +16,7 @@ index 1e8e1d6..5ad928e 100644
      {
        g_propagate_prefixed_error (error, first_error,
 -                                  _("Unable to load /var/lib/dbus/machine-id or /etc/machine-id: "));
-+                                  _("Unable to load @@CONDA_PREFIX@@/var/lib/dbus/machine-id or /etc/machine-id: "));
++                                  _("Unable to load $CONDA_PREFIX/var/lib/dbus/machine-id or /etc/machine-id: "));
```

Comment:
Here's how homebrew does it. https://github.com/Homebrew/legacy-homebrew/blob/21ce7a56cb3fc40fe93d731cc654c69268796772/Library/Formula/glib.rb#L73-L74
That way, it' easier to edit this error message as well.

Reply:
I got a bit sloppy there. I've updated the patch to include the actual prefix in the message here by actually using the C preprocessor macro in the same way as the other cases.